Makefile

```diff
@@ -98,17 +97,17 @@ install.tools: .install.golint .install.govet .install.gitvalidation
 # golint does not even build for <go1.5
 .install.golint:
 ifeq ($(call ALLOWED_GO_VERSION,1.5,$(HOST_GOLANG_VERSION)),true)
-	go get github.com/golang/lint/golint
+	go get -u github.com/golang/lint/golint
 endif
 
 # go vet is now included in >=go1.5, so no need to get it.
 .install.govet:
-ifeq ($(call ALLOWED_GO_VERSION,1.5,$(HOST_GOLANG_VERSION)),true)
-	go get golang.org/x/tools/cmd/vet
+ifneq ($(call ALLOWED_GO_VERSION,1.5,$(HOST_GOLANG_VERSION)),true)
```

Comment:
But we're not testing against 1.4 in Travis.

On Mon, Aug 29, 2016, 15:59 W. Trevor King <notifications@github.com> wrote:

> In Makefile
> <https://github.com/opencontainers/runtime-spec/pull/547#discussion_r76675840>
> :
>
> >  endif
> >
> >  # go vet is now included in >=go1.5, so no need to get it.
> >  .install.govet:
> > -ifeq ($(call ALLOWED_GO_VERSION,1.5,$(HOST_GOLANG_VERSION)),true)
> > -	go get golang.org/x/tools/cmd/vet
> > +ifneq ($(call ALLOWED_GO_VERSION,1.5,$(HOST_GOLANG_VERSION)),true)
>
> This looks like it's allowing 1.4 again. I think we want to check if the
> Go version is in a whitelisted set, or not in a blacklisted set.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/opencontainers/runtime-spec/pull/547/files/a24d96131cdcc41bff4cd64cae01dfe719a79000#r76675840>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAEF6XSyNiLO20rjNI6gDypnullgTJ6Rks5qkzoTgaJpZM4JvfF1>
> .
>